wsd/DocumentBroker.cpp

```diff
@@ -1052,20 +1052,7 @@ bool DocumentBroker::download(
                                  << _storageManager.getLastModifiedTime()
                                  << ", Actual: " << fileInfo.getLastModifiedTime());
 
-            _documentChangedInStorage = true;
-            // Do not reload the document ("close: documentconflict") if there are
-            // any changes in the loaded document, either saved or unsaved.
-            const std::string message =
-                (_lastStorageAttrs.isUserModified() || _currentStorageAttrs.isUserModified() ||
-                 isPossiblyModified())
-                    ? "error: cmd=storage kind=documentconflict"
-                    : "close: documentconflict";
-
-            if (session)
-            {
-                session->sendTextFrame(message);
```

Comment:
I presume that broadcastMessage duplicates the sendTextFrame line here so its loss is ok?

Reply:
Yes. The broadcast already sends to all sessions, including this one. So sending it explicitly here was unnecessary.